frontend/lib/graphql/configured_graphql_provider.dart

```diff
@@ -73,12 +73,7 @@ class ConfiguredGraphQlProviderState extends State<ConfiguredGraphQlProvider> {
         ]),
       ),
     );
-    return GraphQLProvider(
-      client: client,
-      child: CacheProvider(
```

Comment:
why do you remove the cache provider? How improves that caching? :)

Reply:
I guess the CacheProvider once had a meaning, but through upgrades of graphql_flutter, it is a seemingly empty widget. I asked for some clarification on this widget here: https://github.com/zino-hofmann/graphql-flutter/issues/1459. For now I just assumed, that is a leftover class that should be removed in their lib.
Also, I checked whether caching still worked after removing, and it seems to work.